modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -1380,29 +1382,18 @@ mod fedimint_migration_tests {
 
 // Verify the correct Bitcoin RPC is used
 
-#[cfg(feature = "bitcoind")]
 #[test]
 fn verify_bitcoind_backend() {
     let fixtures = fixtures();
     let dyn_bitcoin_rpc = fixtures.dyn_bitcoin_rpc();
     let bitcoin_rpc_kind = dyn_bitcoin_rpc.get_bitcoin_rpc_config().kind;
-    assert_eq!(bitcoin_rpc_kind, "bitcoind")
-}
-
-#[cfg(feature = "esplora")]
-#[test]
-fn verify_esplora_backend() {
-    let fixtures = fixtures();
-    let dyn_bitcoin_rpc = fixtures.dyn_bitcoin_rpc();
-    let bitcoin_rpc_kind = dyn_bitcoin_rpc.get_bitcoin_rpc_config().kind;
-    assert_eq!(bitcoin_rpc_kind, "esplora")
-}
 
-#[cfg(feature = "electrum")]
-#[test]
-fn verify_electrum_backend() {
-    let fixtures = fixtures();
-    let dyn_bitcoin_rpc = fixtures.dyn_bitcoin_rpc();
-    let bitcoin_rpc_kind = dyn_bitcoin_rpc.get_bitcoin_rpc_config().kind;
-    assert_eq!(bitcoin_rpc_kind, "electrum")
+    assert_eq!(
```

Comment:
@bradleystachurski The feature-based detection causes rebuilds on every invocation. I switched it to runtime detection.

Reply:
Oh wow, `cargo-deluxe` is quite useful. Thanks for catching and fixing